### ISSUE TITLE
[Proposal] Заменить спутник ИИ на боксе на спутник прометея

### ISSUE THREAD

**Issue by SkLepP** [TauCetiStation/TauCetiClassic#11315]
#### Подробное описание проблемы
Спутник ИИ на прометее защищен лучше, чем на боксе. На прометее есть дополнительные бластдоры, защищающее само ядро. (а их обходить сложнее, чем ряд стеклянных окон.)
#### Что должно было произойти
Нужно заменить спутник на боксе на более защищенный с прометея
#### Что произошло на самом деле
-
#### Как повторить
-

**Comment by Mercurialaste** [comment]
Предлагаю заменить Бокс на Прометей

**Comment by AndroBetel** [comment]
я думал склепп адекватный

**Comment by Chip11-n** [comment]
До тех пор пока у ИИ нет законов крюзимова, это будет большой ошибкой

**Comment by SkLepP** [comment]
То есть логика: станция с меньшим количеством персонала нуждается в более защищенном ИИ, а станция с бОльшим в менее защищенном ИИ?) Вопрос только в дополнительных бластдорах, это не прям уж дикая имба: спилить до ИИ 2 укрепки или 3, но все же, на боксе такая защита не была бы лишней.

**Comment by AndroBetel** [comment]
на прометее же нет ограничений на персонал никаких
нужно просто принять что на разных картах будут разные условия для игры и не писать такие пропозалы

**Comment by simb11** [comment]
так бокс и прометей предназначены для одного и того же онлайна

**Comment by SkLepP** [comment]
Да, но на деле прометей играют ночью или утром, когда людей мало. Обычно в дневное и вечернее время мы видим только бокс. Мое право предложить)

**Comment by AzzyDreemurr13** [comment]
Честно говоря, спутник на Боксе самый мало защищённый, на Прометее гораздо лучше ситуация. Хотя бы бласт дверки добавить бы во внутреннюю комнату ИИ и вообще заебись. 

И с турелькой что-то сделать, прям чувствуется, что от неё толком нет, за 2 захода олень может вынести эту турельку , и потом самого ИИ на боксе. Неприятная вещь крайне.

**Comment by DarkFurrSss** [comment]
Благослови написавшего этот пропосал. Хотя даже прометеейвский не станет крепостью... Но уже гораздо лучше

**Comment by WispOfKind** [comment]
За мага на прометее сложно украсть наса костюм, если на ИИ сидит чел, который контроллит апц, туррели и еще выпускает холодный воздух из телекомов. Шкафчик с костюмом стоит прямо за турелью, джаммер не позволяет достойно летать по спутнику. 

Если у тебя нет ЕМП, то кража может в лучшем случае закончится двумя бимами в грудь, при условях, что ИИ не выпустит замедляющий холодный воздух и ты используешь что-нибудь заграждающее для туррели, заранее приготовившись к телепорту по свитку
